src/20231113_Multi_IncreaseStablecoinOptimalBorrowRates/AaveV3Metis_IncreaseStablecoinOptimalBorrowRates_20231113.t.sol

```diff
@@ -0,0 +1,77 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import {AaveV3Metis, AaveV3MetisAssets} from 'aave-address-book/AaveV3Metis.sol';
+import {AaveV3PayloadMetis} from 'aave-helpers/v3-config-engine/AaveV3PayloadMetis.sol';
+import {IAaveV3ConfigEngine} from 'aave-helpers/v3-config-engine/IAaveV3ConfigEngine.sol';
+import {GovV3Helpers} from 'aave-helpers/GovV3Helpers.sol';
+
+import 'forge-std/Test.sol';
+import {ProtocolV3TestBase, ReserveConfig} from 'aave-helpers/ProtocolV3TestBase.sol';
+import {AaveV3Metis_IncreaseStablecoinOptimalBorrowRates_20231113} from './AaveV3Metis_IncreaseStablecoinOptimalBorrowRates_20231113.sol';
+contract mock_proposal is AaveV3PayloadMetis {
```

Comment:
why is this mock needed?

Reply:
On metis, all the supply caps and most of the borrows caps are full, making it impossible for e2e to do its job. It was suggested by sakuIstra and ernesto on tg.